README.md

```diff
@@ -1,5 +1,4 @@
-![Ember User Activity](docs/hero-logo.png)
-==========================================
+# ![Ember User Activity](docs/hero-logo.png)
```

Comment:
Why move away from the default syntax provided by the blueprint?

Reply:
There is literally no rendering difference to all these header changes you've made, except that now we will forever have conflicts with the blueprint. :P

Reply:
OOOOOOOHHHHH!  I was just following my markdown linter!  I can revert.